parsec/core/gui/file_items.py

```diff
@@ -70,7 +76,9 @@ def switch_icon(self):
         else:
             self.setToolTip(_("TEXT_FILE_ITEM_IS_NOT_SYNCED_TOOLTIP"))
 
-    def _draw_pixmap(self, source, selected, synced, confined):
+    def _draw_pixmap(
+        self, source: QPixmap | str, selected: bool, synced: bool, confined: bool
```

Comment:
Same thing, is `source` ever a `QPixmap`?

Reply:
Done.